core/data/src/test/java/uk/gov/gchq/gaffer/data/element/function/ToPropertiesTupleTest.java

```diff
@@ -65,6 +65,16 @@ protected ToPropertiesTuple getInstance() {
         return ToPropertiesTuple.class;
     }
 
+    @Override
+    protected Class[] getExpectedSignatureInputClasses() {
+        return new Class[]{Properties.class};
+    }
+
+    @Override
+    protected Class[] getExpectedSignatureOutputClasses() {
+        return new Class[]{};
```

Comment:
I would have expected this to be a PropertiesTuple

Reply:
Now fixed